airbyte-integrations/connectors/source-my-hours/metadata.yaml

```diff
@@ -17,13 +11,30 @@ data:
       enabled: true
     oss:
       enabled: true
+  connectorBuildOptions:
+    # Please update to the latest version of the connector base image.
+    # https://hub.docker.com/r/airbyte/python-connector-base
+    # Please use the full address with sha256 hash to guarantee build reproducibility.
+    baseImage: docker.io/airbyte/python-connector-base:1.0.0@sha256:dd17e347fbda94f7c3abff539be298a65af2d7fc27a307d89297df1081a45c27
+  connectorSubtype: api
+  connectorType: source
+  definitionId: 722ba4bf-06ec-45a4-8dd5-72e4a5cf3903
+  dockerImageTag: 1.0.0
+  dockerRepository: airbyte/source-my-hours
+  githubIssueLabel: source-my-hours
+  icon: my-hours.svg
+  license: MIT
+  name: My Hours
+  releaseDate: 2024-03-18
```

Comment:
@alafanechere is the release date the date the connector was released for the first time? 
If that's the case, we should update this to `2021-12-21`

Reply:
Yes I think so. It's not supposed to be updated on each release.